backend/client-pkg/services/ModelImportService.js

```diff
@@ -0,0 +1,76 @@
+/**
+* @file Defines a function that creates and sends a request to the ModelImportService
+*/
+
+const https = require('https');
+
+/**
+* Forms the body of a proper request to the ModelImportService
+* @param {string} modelId The id of the model image to import
+* @throws Exceptions on invalid parameter types
+* @returns {string} An string that can be used in the path of a request to the ModelImportService
+*/
+const formRequestBody = (modelId) => {
```

Comment:
K sweet yeah I was wondering if I could do string validation the sendRequest. Thanks!